web/src/pages/Settings/Content.tsx

```diff
@@ -1,30 +1,25 @@
 // import {Tabs} from 'antd';
 import DataStore from 'components/Settings/DataStore';
-import {useDataStoreConfig} from 'providers/DataStoreConfig/DataStoreConfig.provider';
 import * as S from './Settings.styled';
 
 /* const TabsKeys = {
   DataStore: 'dataStore',
 }; */
 
-const Content = () => {
-  const {dataStoreConfig} = useDataStoreConfig();
+const Content = () => (
+  <S.Container>
+    <S.Header>
+      <S.Title>Configure Data Store</S.Title>
+    </S.Header>
 
-  return (
-    <S.Container>
-      <S.Header>
-        <S.Title>Configure Data Store</S.Title>
-      </S.Header>
-
-      {/* <S.TabsContainer>
+    {/* <S.TabsContainer>
         <Tabs size="small">
           <Tabs.TabPane key={TabsKeys.DataStore} tab="Configure Data Store"> */}
-      <DataStore dataStoreConfig={dataStoreConfig} />
-      {/* </Tabs.TabPane>
+    <DataStore />
+    {/* </Tabs.TabPane>
```

Comment:
There are commented code in this file sounds to me like a dead code. Does it make sense to remove it?

Reply:
We will introduce this code again after the release. The idea for the release was to include only `DataStores`, but the next epic will include additional settings.